ui/components/ui/deprecated-test-networks/deprecated-test-networks.js

```diff
@@ -45,7 +46,10 @@ export default function DeprecatedTestNetworks() {
             className="deprecated-test-networks__content"
           >
             <Box marginRight={2} color={Color.warningDefault}>
-              <i className="fa fa-info-circle deprecated-test-networks__content__icon" />
+              <Icon
+                name={ICON_NAMES.INFO}
+                className="deprecated-test-networks__content__icon"
```

Comment:
Can probably remove the styles associated with this className

Reply:
Removed